### PR TITLE
Update existing secure conversation welcome screen

### PR DESCRIPTION
**Pull request by gugalo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-3631

**What was solved?**
Designers have requested that small adjustments be made to secure the conversation's welcome screen based on SCv2 project designes.

**Release notes:**

 - [x] Feature: part of SCv2 feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [ ] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to [**Logging from Android SDKs** → **Things to consider for newly added logs**](https://glia.atlassian.net/wiki/spaces/ENG/pages/3568861468/Logging+from+Android+SDKs#Things-to-consider-for-newly-added-logs) in Confluence for more information.

**Screenshots:**
Screenshot_20241022_083240
Screenshot_20241022_083218
Screenshot_20241022_083104

### CODE REVIEW

**Review by DavDo — APPROVED** [review]
LGTM

**Comment by gugalo** [comment]
!squash

**Review by igorkravchenko — APPROVED** [review]

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/messagecenter/ProgressButton.kt [hunk 1]
```diff
@@ -55,7 +57,7 @@ internal class ProgressButton @JvmOverloads constructor(context: Context, attrs:
     private val progressBar get() = binding.progressBar
 
     // End Widgets + Binding
-    private val localeProvider = Dependencies.localeProvider
+    private val localeProvider =  Dependencies.localeProvider
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/messagecenter/ProgressButton.kt:60]: NIT NIT NIT 🙂 
I think extra space is added my a mistake here

**Review by andrews-moc — COMMENTED** [review]
It's out of the scope of this task, probably. I think we should keep the aspect ratio when scaling. Instead of stretching the image. iOS SDK respects the aspect ratio. Do you want me to create a ticket for this improvement?
Screenshot 2024-10-22 at 17 01 32

**Comment by gugalo** [comment]
@andrews-moc Create a bug ticket in backlog
https://glia.atlassian.net/browse/MOB-3732

**Review by andrews-moc — APPROVED** [review]

**Comment by gugalo** [comment]
!squash

**Comment by gugalo** [comment]
!merge

**Pull request merged**